services/reminder.service.js

```diff
@@ -0,0 +1,59 @@
+import {
+  createReminder,
+  fetchReminders,
+  fetchReminder,
+  updateReminder,
+  deleteReminder
+} from '../repository/index.js'
+import responseMessages from '../utils/responseMessages.js'
+
+// Create reminder service logic
+export const createReminderService = async (data) => {
+  try {
+    const { reminderTitle, startDate, endDate, customQuote, challenge, diary } = data
+    await createReminder({ reminderTitle, startDate, endDate, customQuote, challenge, diary })
+    return Promise.resolve(responseMessages.SAVE_SUCCESS)
```

Comment:
This is a service layer. In the service layer should only handle service related things such as calculations and stuff. we don't want to worry about responseMessage in this layer. we can handle it from the controller layer

Reply:
What will we return in this?